### ISSUE TITLE
Constant errors in console

### ISSUE THREAD

**Issue by Janonas** [MrVauxs/PF2e-Target-Damage#26]
The error: "targets is undefined" appears incredibvly often in the sconsole. Also the image for double damadge (the cross) doesn appear the the generic damage buttons (the one with the specific targets appear regularly)

**Comment by MrVauxs** [comment]
Can you show a screenshot of the issues you are facing?

**Comment by Ironxxx** [comment]
I get the following errors currently under target-damage.js, I get the first error every time the modules is used (applies damage):

```
target-damage.js:421 Uncaught TypeError: Cannot read properties of undefined (reading 'options')
[Detected 1 package: pf2e-target-damage]
    at target-damage.js:421:23
```


```
Uncaught TypeError: Cannot read properties of undefined (reading 'length')
[Detected 1 package: pf2e-target-damage]
    at target-damage.js:419:12
```

I get the following under foundry.js

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'startsWith')
[Detected 1 package: pf2e-target-damage]
    at _parseUuid (foundry.js:4176:13)
    at fromUuidSync (foundry.js:4140:49)
    at get token [as token] (target-damage.js:10:10)
    at target-damage.js:273:29

```

**Comment by Janonas** [comment]
![problem](https://user-images.githubusercontent.com/38334265/212555032-a6c93d51-8471-472c-9c9d-bc7a8216699b.jpg)
Here are the errors which happen ecen when it is the only module, also the problem with the X symbol happens when you use the chat portrait module with the target damage module

**Comment by MrVauxs** [comment]
Make sure you are up to date as this is likely fixed now in 2.1.1.

**Comment by Janonas** [comment]
Ah i see that you just did an update which fixed it, Grat work! ill put the corss thing in the new issue
